scripts/Reader.py.experimental

```diff
@@ -125,6 +127,21 @@ class Rdm6300Reader:
         self.rfid_serial.close()
 
 
+class Pn532Reader:
+    def __init__(self):
+        pn532 = Pn532_i2c()
+        self.device = Mifare()
+        self.device.SAMconfigure()
+        self.device.set_max_retries(MIFARE_WAIT_FOR_ENTRY)
+
+    def readCard(self):
+        return str(+int('0x' + self.device.scan_field().hex(),0))
+
+    def cleanup(self):
+        #TODO
+        logger.debug("PN532Reader clean up.")
```

Comment:
Is anything to be done in the cleanup function? Because there's a *TODO*

Reply:
Good catch. Honestly, I don't know if anything needs to be done here for the PN532 reader.

Reply:
I've removed the TODO